schedule-tracking/src/main/java/org/motechproject/scheduletracking/service/ScheduleTrackingService.java

```diff
@@ -8,97 +8,119 @@
 import java.util.List;
 
 /**
- * \ingroup sts
- *
- * Schedule Tracking Service interface
- * Provides methods to enroll an external id into a schedule, fulfill milestones in a schedule
- * and uneroll an external id from a schedule.
- * It also provides querying functionality on Enrollments using various search criteria
- *
+ * Schedule Tracking Service interface provides methods to enroll an external id into a schedule, fulfill milestones
+ * in a schedule and uneroll an external id from a schedule. It also provides querying functionality on Enrollments
+ * using various search criteria.
  */
 public interface ScheduleTrackingService {
+
     /**
-     * Enrolls a user with external id into a schedule using the details in the EnrollmentRequest and schedules alerts. If the user has already enrolled for the same schedule which is currently active, then we update the existing details and reschedule alerts
-     * @param enrollmentRequest
-     * @return enrollmentId string
+     * Enrolls a user with external id into a schedule using the details in the EnrollmentRequest and schedules alerts.
```

Comment:
Regarding my earlier comment about clients and external ids, I guess we can settle for users as the term used throughout the module.